image_explorer/templates/html/image_explorer.html

```diff
@@ -36,6 +37,15 @@ <h2>{{title}}</h2>
                 </div>
             </div>
         {% endfor %}
+        </div>
+        <div class="tutorial-wrapper">
+            <div class="tutorial-container">
+                <h3>Instructions</h3>
+                <p>Click each step in the loop to learn more about the problem solving approach. When you are finished, click the forward arrow to continue.</p>
+                <div class="tutorial-video">&nbsp;</div>
+                <a href="javascript:void(0)" id="hide-me">Start Excercise</a>
```

Comment:
Does this link need the `href` attribute? I think if we leave this off it should have the same effect were there no javascript handler.

Also could we change the ID for this to be a bit more specific. Maybe something like `#hide-tutorial`, or `#hide-instructions` ?

Reply:
Thanks @kaizoku for the prompt review. I have removed the "href" attribute and changed the ID to "hide-tutorial"

Can you please merge the code as I don't have the rights?

Reply:
@kaizoku Please wait as I have got some feedback from Olivia and I have to do some tweaks in colors. I will let you know when to merge the code. Thanks